multi-arch-builders/coreos-s390x-rhcos-builder.bu

```diff
@@ -0,0 +1,103 @@
+# This butane config will do the following:
+#
+# - Merge in the builder-common.ign Ignition file
+# - Allow the builder user to log in with the associated ssh key
+# - Set a hostname
+# - Create and initialize the secex-data volume
```

Comment:
This needs updating:

```suggestion
# - Merge in the coreos-s390x-builder.ign Ignition file
# - Create and initialize the secex-data volume
```

Might have to be split across the two commits you have in this PR.

Reply:
I updated the description accordingly and split it between the 2 commits